spec/requests/api/v1/driver_commission_histories_spec.rb

```diff
@@ -1,24 +1,64 @@
 require 'rails_helper'
 
 RSpec.describe '/api/v1/all', :type => :request do
-  let(:all) do
-    DriverCommissionHistoryAdapter.all
-  end
+  describe 'all' do
+    let(:all) do
+      DriverCommissionHistoryAdapter.all
+    end
 
-  let(:by_paid_date) do
-    DriverCommissionHistoryAdapter.by_paid_date(Date.new(2012, 01, 01))
-  end
+    let(:all_url) { '/api/v1/driver_commissions_history' }
+
+    context 'when application id and secret are valid' do
```

Comment:
Would this not be redundant given that we did the same test in the controller tests?